.github/workflows/_ci.yaml

```diff
@@ -106,12 +63,7 @@ jobs:
       ARCHITECTURE: ${{ inputs.ARCHITECTURE }}
       BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
       BASE_IMAGE: ${{ needs.build-base.outputs.DOCKER_TAG }}
-      REPO_JAX: ${{ needs.metadata.outputs.REPO_JAX }}
-      REF_JAX: ${{ needs.metadata.outputs.REF_JAX }}
-      REPO_XLA: ${{ needs.metadata.outputs.REPO_XLA }}
-      REF_XLA: ${{ needs.metadata.outputs.REF_XLA }}
-      REPO_TE: ${{ needs.metadata.outputs.REPO_TE }}
-      REF_TE: ${{ needs.metadata.outputs.REF_TE }}
+      TRIAL_BRANCH: ${{ inputs.TRIAL_BRANCH }}
```

Comment:
It's not provided to the dockerfile. You're looking at `_build_jax.yaml`'s `workflow_call.inputs`. 

The reason it's passed is because within `_build_jax.yaml` is the `actions/checkout@v3` and I want to checkout the state of the manifest+patches on the trial branch (see [this](https://github.com/NVIDIA/JAX-Toolbox/pull/405/files/0828455c4d9478eba642ab24a8d17140b84c51d9#diff-4e3cbf29dd2e5a1bbae7236b97ab898d70d297ee50fba4765d4a71b9b8696694R92)).

This solves the problem that occurs when building the Nightly builds off of the `main` branch, and we need to build from "staged" changes, which is accomplished by checking out the "trial branch".